src/components/form/inputs/NormalButton/NormalButton.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { ComponentMeta, StoryFn } from '@/utils/storybook';
+import { NormalButton } from './NormalButton.component';
+
+const NormalButtonStory: ComponentMeta<typeof NormalButton> = {
+  component: NormalButton,
+};
+
+export default NormalButtonStory;
+
+const Template: StoryFn<typeof NormalButton> = args => (
+  <NormalButton {...args} />
+);
+
+export const Contained = Template.bind({});
+Contained.args = {
+  label: 'Normal',
+};
```

Comment:
先のレビューで漏れたんですが、各argsの中に`disabled: false`と入れておきましょうか

```suggestion
  disabled: false;
};
```

Reply:
<img width="448" alt="image" src="https://github.com/ih14c-13-14/artist-frontend/assets/49822810/c422d0d3-78a3-4536-b5f1-9e55789e0028">
こういうトグルが生えてきます

Reply:
追加したので `Controls` の項目に `disable` のトグルスイッチが出てきたはず